build-tools/packages/build-cli/docs/release.md

```diff
@@ -203,9 +204,14 @@ FLAGS
 
   --baseFileName=<value>
       If provided, the output files will be named using this base name followed by the report kind (caret, simple, full,
-      tilde) and the .json extension. For example, if baseFileName is 'foo', the output files will be named
+      tilde, legacy-compat) and the .json extension. For example, if baseFileName is 'foo', the output files will be named
       'foo.caret.json', 'foo.simple.json', etc.
 
+  --compatVersionInterval=<value>
```

Comment:
I'm not clear about the reason this value needs to configurable in the CLI. Could we remove the flag for now, and add if/when we have a real need? If that need is real now, can you explain it to me?